### DISCUSSION TITLE
Console Guidance for EOL Patterns?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2447] [Q&A] [answered]
Hello! I am looking for any guidance the console team has for announcing in console that we are "end of life-ing" a product? I'm assuming I will use components like modals, warning toasts, but does the console team have any guidance on in product patterns for EOL? If not thats okay - but figured id check.

**Comment by loreina** [comment, marked as answer]
Hi @mjanssen2021!

Unfortunately, Paste doesn't currently have any guidance or pattern on EOL. I'd recommend syncing with @bhamtil-twilio and @kendal-kirby to see if there are any Console-specific guidelines for this. Thanks!

**Comment by mjanssen2021** [comment]
Thank you @loreina !! Ill ask them.
